verible/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+set -x
+
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
```

Comment:
A build.sh receipe should never call `sudo`

Reply:
I actually copied these lines from the Slang recipe. I guess we should it remove there too then